src/components/Dissolution/CompleteResolution.vue

```diff
@@ -781,13 +787,33 @@ ul {
   }
 }
 
-::v-deep #confirm-resolution-section {
+::v-deep {
   // override default validation styling so checkbox does not turn red on validation error
-  .v-input--selection-controls__input .error--text {
+  .v-input--selection-controls__input .error--text,
+  .v-messages__message,
+  .v-input__slot {
+    color: $app-lt-gray;
+  }
+  .v-input__icon .error--text {
     color: $app-lt-gray !important;
   }
 }
 
+.invalid-section ::v-deep {
```

Comment:
Comment, please.

Reply:
Hey Alex, please let me resolve the conversations that I've started. These act like a "todo list" for me so that I know what is outstanding to re-review. If you close them then I may forget to check them.